steps/create-hive-tables.py

```diff
@@ -1,11 +1,7 @@
 import boto3
 import ast
 from logger import setup_logging
-
-level = "info"
-logger_path = "/var/log/adg/hive_tables_creation_log.log"
-logger_format = "{ 'timestamp': '%(asctime)s', 'log_level': '%(levelname)s', 'message': '%(message)s' }"
-the_logger = setup_logging(level, logger_path , logger_format)
+the_logger = setup_logging("info", log_path = "/var/log/adg/hive_tables_creation_log")
```

Comment:
log levels should come from the environment, not hard coded. Is there a reason for doing this?

Reply:
good feedback thanks Dan, i have added this in!